src/SDK/Common/Dev/Compatibility/BC/InstrumentationLibraryInterface.php

```diff
@@ -13,7 +13,7 @@ interface InstrumentationLibraryInterface extends Moved
 /**
  * @codeCoverageIgnoreStart
  */
-class_alias(Moved::class, 'OpenTelemetry\SDK\InstrumentationLibraryInterface');
+class_alias(InstrumentationLibraryInterface::class, 'OpenTelemetry\SDK\InstrumentationLibraryInterface');
```

Comment:
This should point to `'OpenTelemetry\SDK\Common\Instrumentation\InstrumentationLibraryInterface'` as this was the most recent change. 
```php
class_alias(InstrumentationLibraryInterface::class, 'OpenTelemetry\SDK\Common\Instrumentation\InstrumentationLibraryInterface');
```

Reply:
Ah, ok, I was going for BC to the original location. Lucky I checked with you.
```suggestion
class_alias(InstrumentationLibraryInterface::class, 'OpenTelemetry\SDK\Common\Instrumentation\InstrumentationLibraryInterface');
```